explainaboard/tasks.py

```diff
@@ -130,7 +134,7 @@ class TaskCategory:
         ],
     ),
     TaskCategory(
-        "kg-link-tail-prediction",
+        "kg_prediction",
```

Comment:
I know this is WIP, but I was looking at it out of curiosity and noticed that this name uses underbars while other names use hyphens. This was also the case in DataLab as well. Maybe we can consolidate to hyphens?

Reply:
Yes, this is a typo, and thanks for capturing it!